README.md

```diff
@@ -44,6 +44,7 @@ Platform      |    Status     |
 -----------     | --------------|
 Arduino         | [![Arduino](https://github.com/tensorflow/tflite-micro-arduino-examples/actions/workflows/ci.yml/badge.svg?event=schedule)](https://github.com/tensorflow/tflite-micro-arduino-examples/actions/workflows/ci.yml) [![Antmicro](https://github.com/antmicro/tensorflow-arduino-examples/actions/workflows/test_examples.yml/badge.svg)](https://github.com/antmicro/tensorflow-arduino-examples/actions/workflows/test_examples.yml) |
 ESP32           | [![ESP32](https://github.com/espressif/tflite-micro-esp-examples/actions/workflows/build.yml/badge.svg)](https://github.com/espressif/tflite-micro-esp-examples/actions/workflows/build.yml) |
+Renesas EK-RA6M4 | [TFLM Examples for Renesas EK-RA6M4](https://github.com/renesas/tflite-micro-renesas) 
```

Comment:
@ES-Renesas

Reply:
Hello Deqiang, 
Thank you for doing the PR, as we are planning to add more boards with TFLM examples, can the table say 
Renesas | [TFLM examples for Renesas boards] (https://github.com/renesas/tflite-micro-renesas)

Thank you :)

Reply:
done